src/interfaces/payment-interfaces/fees-interface.ts

```diff
@@ -0,0 +1,14 @@
+/** Interface for Fees from Payment API. */
+import { TaxesIF } from '@/interfaces'
+
+export interface FeesIF {
```

Comment:
If some of these are optionally returned by the network call, you could specify them as optional in the interface like this:
`futureEffectiveFees?: number`
LMK if any of these is optional. If not then I will resolve this comment.

Reply:
Ok. I did it based on the schema:
https://github.com/bcgov/sbc-pay/blob/development/pay-api/src/pay_api/schemas/schemas/fees.json